code/game/atoms.dm

```diff
@@ -270,6 +276,9 @@
 /atom/proc/emp_act(severity)
 	return
 
+/atom/proc/water_act(volume, temperature, source, method = REAGENT_TOUCH) //amount of water acting : temperature of water in kelvin : object that called it (for shennagins)
```

Comment:
[DMDOC compliance](https://github.com/ParadiseSS13/Paradise/blob/master/.github/AUTODOC_GUIDE.md) would be welcome